moto/core/responses.py

```diff
@@ -253,8 +253,15 @@ def setup_class(
                 querystring[key] = [value]
 
         raw_body = self.body
+
         if isinstance(self.body, bytes) and not use_raw_body:
-            self.body = self.body.decode("utf-8")
+            try:
+                self.body = self.body.decode("utf-8")
+            except:
+                try
```

Comment:
```suggestion
                try:
```

Reply:
The base64 change is to support some URL encodings that are used when the snapshot data come in this form. Not sure if it really matters since payload is not used AFAIK. The real fix is to just handle the decode exception. I'll change to using the try try pattern.